framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -893,6 +990,7 @@ def launch_simulation(self, launch_mode, **data):
 
         burst_name = current_form.simulation_name.value
         session_stored_simulator = common.get_from_session(common.KEY_SIMULATOR_CONFIG)
+        session_stored_simulator.simulation_length = float(data['simulation_length'])
```

Comment:
Yes, it can, I fixed this. The problem was the in the SimulatorFinalFragment, the name of the field was length, not simulation_length, so it wasn't added by the fill_from_post. Now it is okay, I renamed it to simulation_length.